test/transcoding/enqueue_kernel.cl

```diff
@@ -115,15 +122,15 @@ kernel void device_side_enqueue(global int *a, global int *b, int i, char c0) {
 
   // Emits global block literal and block kernel.
 
-  // CHECK-SPIRV: PtrAccessChain [[Int32LocPtrTy]] [[LocalBuf41:[0-9]+]]
+  // CHECK-SPIRV-COUNT-4: PtrAccessChain [[Int32LocPtrTy]] [[LocalBuf41:[0-9]+]]
   // CHECK-SPIRV: PtrAccessChain [[Int32LocPtrTy]] [[LocalBuf42:[0-9]+]]
   // CHECK-SPIRV: PtrAccessChain [[Int32LocPtrTy]] [[LocalBuf43:[0-9]+]]
   // CHECK-SPIRV: PtrCastToGeneric {{[0-9]+}} [[BlockLit4Tmp:[0-9]+]] [[BlockGlb2:[0-9]+]]
   // CHECK-SPIRV: Bitcast [[Int8PtrGenTy]] [[BlockLit4:[0-9]+]] [[BlockLit4Tmp]]
-  // CHECK-SPIRV: EnqueueKernel [[Int32Ty]] {{[0-9]+}} {{[0-9]+}} {{[0-9]+}} {{[0-9]+}}
-  //                            [[ConstInt0]] [[EventNull]] [[EventNull]]
-  //                            [[BlockKer4]] [[BlockLit4]] [[ConstInt8]] [[ConstInt8]]
-  //                            [[LocalBuf41]] [[LocalBuf42]] [[LocalBuf43]]
+  // CHECK-SPIRV: EnqueueKernel [[Int32Ty]] [[#]] [[#]] [[#]] [[#]]
+  // CHECK-SPIRV-SAME: [[ConstInt0]] [[#]] [[#]]
+  // CHECK-SPIRV-SAME: [[#InvokeFunc4]] [[BlockLit4]] [[ConstInt12]] [[ConstInt8]]
```

Comment:
`ConstInt8` becoming `ConstInt12` could be the main concern. However the check was not working for so long, that I'd like to assume that it's a correct "new" compiler behavior.

Reply:
Do we know, where `Param Size` is being calculated?

Reply:
Found that value is generated by us. Maybe layout has been changed since the test was written?
https://github.com/KhronosGroup/SPIRV-LLVM-Translator/blob/93fb018aa342da185ef7f944b27d6b5df104d97b/lib/SPIRV/OCLToSPIRV.cpp#L1508-L1513